src/docs/environment/index.mdx

```diff
@@ -112,7 +112,9 @@ Now, if you try and run `volta`, you should see some help text, meaning volta is
 node -v
 ```
 
-Volta intercepts this and automatically downloads and installs the node version in sentry's `package.json`.
+Volta intercepts this and automatically downloads and installs the node and yarn versions in sentry's `package.json`.
+
+Note that if you didn't already have node and yarn installed on your computer prior to this step, you'll need to run `volta install node@x.x.x` and `volta install yarn@x.x.x` when outside the `sentry` directory in order to use `node` and `yarn` elsewhere.
```

Comment:
```suggestion
```

Reply:
Same thing here, why does anyone need to care about this. Its not related to Sentry and this isnt generic "how to use node" or "how to use Volta" docs.